r/README.md

```diff
@@ -23,7 +24,17 @@ Windows or macOS.
 install.packages("arrow")
 ```
 
-Alternatively, if you are using conda you can install arrow from conda-forge:
+Alternatively, you can install the development version of arrow from R-universe.
+
+```r
+# Binaries for MacOS/Windows
+install.packages("arrow", repos = "https://apache.r-universe.dev")
```

Comment:
This command should fail because the repository (`https://apache.r-universe.dev`) does not contain any dependencies for the arrow package.

```suggestion
install.packages('arrow', repos = c('https://apache.r-universe.dev', 'https://cloud.r-project.org'))
```